tests/Tests/Payload/SearchAllPayloadResponseTest.php

```diff
@@ -41,10 +41,10 @@ protected function assertResponse(array $responseData, PayloadResponseInterface
         $fileResult = $payloadResponse->getFileResult();
         $messageResult = $payloadResponse->getMessageResult();
 
-        $this->assertInstanceOf('CL\Slack\Model\FileResult', $fileResult);
-        $this->assertInstanceOf('CL\Slack\Model\MessageResult', $messageResult);
-        $this->assertCount(1, $fileResult->getMatches());
-        $this->assertCount(1, $messageResult->getMatches());
+        self::assertInstanceOf('CL\Slack\Model\FileResult', $fileResult);
+        self::assertInstanceOf('CL\Slack\Model\MessageResult', $messageResult);
```

Comment:
For consistency, you might want to use the `...::class` here too instead of the string?
Or change it back e.g. in tests/Tests/Payload/AbstractSearchPayloadResponseTest.php etc.

```suggestion
        self::assertInstanceOf(CL\Slack\Model\FileResult::class, $fileResult);
        self::assertInstanceOf(CL\Slack\Model\MessageResult::class, $messageResult);
```